compliance/features/synthetic-source.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Synthetic source
+  Support synthetic source
+
+  @3.2.0
+  Scenario: Installer leverages source true
+   Given the "logs_synthetic_mode" package is installed
+     And a policy is created with "logs_synthetic_mode" package
+    Then index template "logs-logs_synthetic_mode.synthetic" has a field "decision_id" with "store:true"
```

Comment:
I thought as it required changes in Fleet, this test would fail

Reply:
Ok, I think as it requres spec 3.2.0, this complicance test is not tested:
https://github.com/elastic/package-spec/blob/a32f97206dccb24ff6343028d9ef54898b585f75/.buildkite/pipeline.trigger.compliance.tests.sh#L29-L31

Should we take the opportunity to set 8.13.4 as Elastic stack version for the spec version 3.1.0? 

Even if we add there `compliance_test 8.14.0-SNAPSHOT 3.2.0`, this test would fail because of the lack of support, am I right ?

Reply:
> Should we take the opportunity to set 8.13.4 as Elastic stack version for the spec version 3.1.0?

Or I can change to 8.14.0-SNAPSHOT, yes.

> Even if we add there compliance_test 8.14.0-SNAPSHOT 3.2.0, this test would fail because of the lack of support, am I right ?

Correct, we cannot enable 3.2.0 yet.